CHANGELOG.md

```diff
@@ -41,87 +41,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-<<<<<<< HEAD
 ## [v7.3.0](https://github.com/osmosis-labs/osmosis/releases/tag/v7.3.0)
 
 ### Bug Fixes
 
 * [1390](https://github.com/osmosis-labs/osmosis/pull/1390) Upgrade sdk to v0.45.0x-osmo-v7.9, fixing data races with GRPC
```

Comment:
Do we want to have a changelog entry here?

Reply:
It is added here right? https://github.com/osmosis-labs/osmosis/pull/2261/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR42-R45